homeassistant/components/light/__init__.py

```diff
@@ -1251,3 +1323,10 @@ def supported_features_compat(self) -> LightEntityFeature:
             report_issue,
         )
         return new_features
+
+    def __should_report_light_issue(self) -> bool:
+        """Return if light color mode issues should be reported."""
+        if not self.platform:
+            return True
+        # philips_js and zha have known issues, we don't need users to open issues
+        return self.platform.platform_name not in {"philips_js", "zha"}
```

Comment:
This return condition is not validated in the tests.  Consider adding platform_name to one of the parametrized tests?

Are we better off having these two integrations explicitly opt out of the issue, rather than coding here?  This way when the issue is fixed in the integration no changes are required to this base class?

Reply:
> This return condition is not validated in the tests. Consider adding platform_name to one of the parametrized tests

Hmm, you're right 👍 I'll update the tests.

> Are we better off having these two integrations explicitly opt out of the issue, rather than coding here

We usually use this pattern to temporarily opt-out of validation, but I guess the integrations could opt out too.

Reply:
Test update looks good.

Since this is the established pattern that is fine with me, as consistency is important.  The risk is if this does not get removed after zha fixes the issue, it could mask the issue being in zha (e.g. it wasn't fixed completely).  I'm fine with it as is.